### PR TITLE
Add simple api trading script

### PR DESCRIPTION
**Pull request by ghubstan**
Emulates Bob and Alice trading regtest BTC with each other with.

The script output is intended to be a tutorial, not the script itself.

https://github.com/bisq-network/bisq/pull/5081 has to merged before this one.

### CODE REVIEW

**Comment by ghubstan** [comment]
Since the 1st check-in of these two scripts (commit d756d3d), several more commits were pushed, making a PR review seem more complex than it should be.  These scripts have never been reviewed before, so I recommend any reviewer look at the latest version of the entire file(s) instead of individual commits.

File: apitest/scripts/trading-script-env.sh [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+		DUMMY_ACCT=$DUMMY_ACCT_1
+	else
+		DUMMY_ACCT=$DUMMY_ACCT_2
+	fi
+	ACCT_ID=$(echo -e $DUMMY_ACCT | awk '{print $NF}')
+	echo "${ACCT_ID}"
+}
+
+createoffer() {
+ 	CREATE_OFFER_CMD=$1
```
Comment (sqrrm) [inline apitest/scripts/trading-script-env.sh]: Indent

File: apitest/scripts/trading-script-env.sh [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    OFFER_DESC=$($CREATE_OFFER_CMD)
+
+    ### This is an effort to handling any createoffer error.  If error, should echo Error, and the calling script
+    ### should sleep awhile then exit 1.
+	if [[ "$OFFER_DESC" != "Buy/Sell"* ]]; then
```
Comment (sqrrm) [inline apitest/scripts/trading-script-env.sh]: Indent

File: apitest/scripts/trading-script-env.sh [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+genbtcblock() {
+    # TODO use bitcoin-cli to get new address; cannot assume same address exists on other regtest bitcoin-core hosts.
+    CMD='bitcoin-cli -regtest -rpcport=19443 -rpcuser=apitest -rpcpassword=apitest generatetoaddress 1 "2N5J6MyjAsWnashimGiNwoRzUXThsQzRmbv"'
+    printcmd "$CMD"
+    bitcoin-cli -regtest -rpcport=19443 -rpcuser=apitest -rpcpassword=apitest generatetoaddress 1 "2N5J6MyjAsWnashimGiNwoRzUXThsQzRmbv"
+}
+
+escapepluschar() {
+ 	STRING=$1
+	NEW_STRING=$(sed 's/+/\\&/g' <<< ${STRING})
```
Comment (sqrrm) [inline apitest/scripts/trading-script-env.sh]: Seems there is inconsistent usage of tab vs space for indentation

**Review by sqrrm — COMMENTED** [review]
I think it would be good to cancel stop the script if it fails at any step. My first time running Bob couldn't find any offers to take but still continued to try to take it. Just looks a bit odd that the script continues after it fails one step that it later depends on.

**Comment by ghubstan** [comment]
Closing.  There are too many changes to come, including re-defining the script opt labels, and creating payment accts for trading in a variety of country currencies.  Will re-create a cleaner PR when ready.

**Pull request closed without merging**